.env

```diff
@@ -1 +1,2 @@
 REACT_APP_GOOGLE_CLIENT_ID=706375540729-sqnnig7o0d0uqmvav0h8nh8aft6l55u3.apps.googleusercontent.com
+API_BASE=https://hydrant.xvm.mit.edu
```

Comment:
We probably need to use a separate xvm instance to do the server properly, unless we want to implement it all in CGI. @psvenk thoughts?